proposal/indented-syntax-improvements.md

```diff
@@ -1,25 +1,23 @@
-# Indented Syntax Improvements
+# Indented Syntax Improvements: Draft 1.1
 
 *([Issue](https://github.com/sass/sass/issues/216))*
```

Comment:
Please link to the changelog here as well.

Reply:
Added!